core/src/test/java/com/google/googlejavaformat/java/FormatterIntegrationTest.java

```diff
@@ -50,7 +50,7 @@
   private static final ImmutableSet<String> JAVA14_TESTS =
       ImmutableSet.of("I477", "Records", "RSLs", "Var", "ExpressionSwitch", "I574", "I594");
 
-  private static final ImmutableSet<String> JAVA16_TESTS = ImmutableSet.of("I588");
+  private static final ImmutableSet<String> JAVA16_TESTS = ImmutableSet.of("I588", "I603");
```

Comment:
Can you add a `JAVA15_TESTS` entry for the new tests?

Reply:
That makes sense, I’ll fix that.